synfig-studio/src/gui/mainwindow.cpp

```diff
@@ -239,6 +243,14 @@ MainWindow::toggle_show_menubar()
 		menubar->hide();
 }
 
+void
+MainWindow::toggle_show_toolbar()
+{
+	App::enable_mainwin_toolbar = !App::enable_mainwin_toolbar;
+	if (etl::loose_handle<CanvasView> canvas_view = App::get_selected_canvas_view())
+		canvas_view->toogle_displaybar();
```

Comment:
I suggest to rename `CanvasView::toogle_displaybar()` to `CanvasView::toggle_show_toolbar()`
(to make it correspond with `MainWindow::toggle_show_toolbar()` above).